bookshelf-standard/2-structured-data/README.md

```diff
@@ -0,0 +1,22 @@
+# Bookshelf App for Java on App Engine Standard Tutorial
```

Comment:
Ah - yeah. Mostly I just copied it from the flex bookshelf and added "Standard" ;-) But also - I figure, most of the explanatory text will be on the docsite, so probably best to avoid duplication.